lib/global_registry/base.rb

```diff
@@ -23,6 +24,24 @@ def self.get(params = {}, headers = {})
     def get(params = {}, headers = {})
       request(:get, params, nil, headers)
     end
+    def self.get_all_pages(params = {}, headers = {})
```

Comment:
Yeah, I guess it could, but the reason I wanted it as a class-level method is that I could use it for something like `Subscription.all` and have that return a list of all the subscription details. If it were instance-level, I would need to call the method as `Subscription.new.all` which I'm open to but looks a little less clean to me. Thoughts?